sdks/java/io/google-cloud-platform/build.gradle

```diff
@@ -20,7 +20,7 @@ import groovy.json.JsonOutput
 
 plugins { id 'org.apache.beam.module' }
 applyJavaNature(
-
+  //enableStrictDependencies:true,
```

Comment:
If I enableStrictDependencies here,I am getting:
```
* What went wrong:
Execution failed for task ':sdks:java:io:google-cloud-platform:analyzeClassesDependencies'.
> No signature of method: ca.cutterslade.gradle.analyze.AnalyzeDependenciesTask$_action_closure3$_closure8.doCall() is applicable for argument types: (org.gradle.api.internal.artifacts.PreResolvedResolvableArtifact) values: [beam-runners-core-java.jar (project :runners:core-java)]
  Possible solutions: findAll(), findAll(), isCase(java.lang.Object), isCase(java.lang.Object)

```

Reply:
I ran into this as well. I was able to avoid it temporarily by downgrading from Gradle 6.8 to 6.7, however we will need to figure out a better solution. The first step is to figure out where the `:runners:core-java` dependency is coming from, since it isn't explicitly declared here.